README.md

```diff
@@ -26,4 +27,6 @@ Follow instructions here https://cloud.google.com/cloud-build/docs/configure-thi
 | GCLOUD_PROJECT_ID                     | no default                             | GCP project id                                                                                       |
 | SLACK_WEBHOOK_URL                     | no default                             | Slack Webhook URL. Read more https://api.slack.com/incoming-webhooks                                 |
 | GCLOUD_PUBSUB_SUBSCRIPTION_NAME       | cloudbuild-notifier-subscription       | Google Cloud Pub/Sub topic subscription. Read more: https://cloud.google.com/pubsub/docs/subscriber  |
-
+| STATUS_TO_FILTER                     | All statuses                             | A list of build status that should trigger notifications to be sent to Slack e.g. ["FAILURE", "INTERNAL_ERROR"]. By default, notifications will be sent for all statuses |
+| BRANCH_TO_FILTER                     | All branches                             | A list of branches whose builds should trigger notifications to be sent to Slack e.g. ["main", "production"]. By default, notifications will be sent for all branches |
+| SOURCE_TO_FILTER                     | Any source                             | A substring that will be used to match which source Repos should trigger notifications to be sent to Slack e.g. "org-name/repo-name" |
```

Comment:
Does setting this imply only a single source may be used?  What happens if we want different branch filters based on different sources? Or a combination of statuses by branch by source?  I don't want to make this more complicated than necessary, but I want to make sure we're designing something that supports user-expected cases.

Reply:
I'll change this to use a list of dictionaries to determine the various combinations of statuses by branch by source:
e.g. [
       {
           "status": ["FAILURE", "INTERNAL_ERROR"],
           "branches": ["main", "production"],
           "sources": ["org-name/repo-name1", "org-name/repo-name2"]
       }
       ]
That should allow for any combinations of status, branches and sources